kafka-ui-react-app/src/components/Topics/Topic/Edit/Edit.tsx

```diff
@@ -53,28 +53,6 @@ export const DEFAULTS = {
   maxMessageBytes: 1000012,
 };
 
-const topicParams = (topic: TopicWithDetailedInfo | undefined) => {
-  if (!topic) {
-    return DEFAULTS;
-  }
-
-  const { name, replicationFactor } = topic;
-
-  return {
-    ...DEFAULTS,
-    name,
-    partitions: topic.partitionCount || DEFAULTS.partitions,
-    replicationFactor,
-    [TOPIC_CUSTOM_PARAMS_PREFIX]: topic.config
-      ?.filter(
-        (el) =>
-          el.value !== el.defaultValue &&
-          Object.keys(TOPIC_CUSTOM_PARAMS).includes(el.name)
-      )
-      .map((el) => ({ name: el.name, value: el.value })),
-  };
-};
-
 let formInit = false;
```

Comment:
I'm not sure why this is here?  I can see you reassign it `formInit = false;` on line 79 inside useEffect. This is not a good way to do, instead use useState or if you don't need to trigger rerenders, use useRef to store this variable.